content/docs/sidebar.json

```diff
@@ -102,11 +102,11 @@
           },
           {
             "label": "dvc.yaml & dvc.lock",
-            "slug": "dvc.yaml"
+            "slug": "dvc-yaml"
```

Comment:
Hi @rogermparent what do you think of this little bug? 🙂 

Expected I guess. Or is it something we should be able to support to have URLs ending in .anything and have them work when loaded directly? (In this case https://dvc.org/doc/user-guide/dvc-files/dvc.yaml doesn't load directly, but works once the interface is loaded on another URL e.g. from https://dvc.org/doc/user-guide)

Mainly curious, for now we'll just avoid these unconventional URLs.

Reply:
@jorgeorpinel please, create a ticket, let's not focus on this at the moment